starter-code/src/functions-and-arrays.js

```diff
@@ -1,52 +1,47 @@
 // Find the maximum
+function maxOfTwoNumbers (num1, num2) {
+  if (num1 > num2) {
+    return num1;
+  } else {
+    return num2;
+  };
+};
 
 // Finding Longest Word
-var words = [
-  'mystery',
-  'brother',
-  'aviator',
-  'crocodile',
-  'pearl',
-  'orchard',
-  'crackpot'
-];
+function findLongestWord (words) {
+  var result = '';
+  for (var ix = 0; ix < words.length; ix++){
+    if (words[ix].length > result.length){
+       result = words[ix];
+    }
+  }
+  return result;
+}
 
 // Calculating a Sum
 
-var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
+function sumArray (array) {
+  var sum = 0;
+  for (var ix = 0; ix < numbers.length; ix++){
+    sum += numbers[ix];  
+  }
+  return sum;
+}
+    
 
 // Calculate the Average
+function averageNumbers (numbers) {
+  const numbers = [2, 6, 9, 10, 7, 4, 1, 9].reduce((a, b) => a + b, 0);
```

Comment:
Here you could have just called the sumArray function you have above and pass it the numbers. You over complicated a bit here. There's a log going on in that reduce